app/controllers/api/host_aggregates_controller.rb

```diff
@@ -25,6 +25,32 @@ def delete_resource_main_action(_type, host_aggregate, _data = {})
       {:task_id => host_aggregate.delete_aggregate_queue(current_user.userid)}
     end
 
+    def add_host_resource(type, id, data = {})
+      raise if id.nil? || data["host_id"].nil?
+
+      host_aggregate = resource_search(id, type)
+      raise unless host_aggregate.supports?(:add_host)
+
+      new_host = resource_search(data["host_id"], :hosts)
+      task_id = host_aggregate.add_host_queue(current_user.userid, new_host)
+      action_result(true, "Adding #{host_aggregate_ident(host_aggregate)}", :task_id => task_id)
+    rescue => err
+      action_result(false, err.to_s)
+    end
+
+    def remove_host_resource(type, id, data = {})
+      raise if id.nil? || data["host_id"].nil?
+
+      host_aggregate = resource_search(id, type)
+      raise unless host_aggregate.supports?(:remove_host)
+
+      new_host = resource_search(data["host_id"], :hosts)
+      task_id = host_aggregate.remove_host_queue(current_user.userid, new_host)
+      action_result(true, "removing #{host_aggregate_ident(host_aggregate)}", :task_id => task_id)
```

Comment:
Most of the action results start with a capital letter

```suggestion
      action_result(true, "Removing #{host_aggregate_ident(host_aggregate)}", :task_id => task_id)
```